packages/storage/package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "build": "rollup -c rollup.config.compat.js && yarn build:exp",
-    "build:exp": "rollup -c rollup.config.exp.js ; yarn api-report",
+    "build:exp": "rollup -c rollup.config.exp.js && yarn api-report",
```

Comment:
I think we want `;` which executes regardless of the return status of the previous command. I believe `rollup -c rollup.config.exp.js` will return an error status in the exp release build, though build will go through.

Reply:
Oh, ok, restored.